.gitlab-ci/packet.yml

```diff
@@ -68,6 +68,11 @@ packet_ubuntu18-flannel-containerd:
   extends: .packet
   when: manual
 
+packet_ubuntu18-flannel-containerd-once:
```

Comment:
why the word once?

Reply:
this is short for variable name `download_run_once`
I can change for any word that you find more appropriate